src/renderer/src/lib/locale/en/errors.json

```diff
@@ -12,6 +13,7 @@
 	"noDevhubAuthenticationTokenFound": "No authentication token found for devhub",
 	"noPublishersAvailableError": "No publishers available",
 	"noRunningHolochainManagerError": "No running Holochain manager",
+	"remoteCallFailedError": "Remote call failed. Unable to fetch happ bytes due to no available peers at the moment. Please try again later.",
```

Comment:
This is not correct actually. If the error say that it failed for all available hosts then there were peers available but they weren't able to deliver the data for some reason. I think we need to actually discern the different scenarios and return specific errors for each case. I also need to look into why the error returns an empty array because it shouldn't.

Reply:
can we make a generic message for now like

```
"remoteOperationFailedError": "Remote operation failed. Please try again later."
```

and handle more specific errors later?